### PR TITLE
[needs-docs] Labels toolbar actions no longer need the layer to be in…

### PR DESCRIPTION
**Pull request by DelazJ**
… edit mode

## Checklist

- [ ] Commit messages are descriptive and explain the rationale for changes
- [ ] Commits which fix bugs include `Fixes #11111` at the bottom of the commit message
- [ ] I have read the [QGIS Coding Standards](https://docs.qgis.org/testing/en/docs/developers_guide/codingstandards.html) and this PR complies with them
- [ ] New unit tests have been added for core changes
- [ ] I have run [the `scripts/prepare-commit.sh` script](https://github.com/qgis/QGIS/blob/master/.github/CONTRIBUTING.md#contributing-to-qgis) before each commit
- [ ] I have evaluated whether it is appropriate for this PR to be backported, backport requests are left as label or comment

### CODE REVIEW

**Review by nyalldawson — APPROVED** [review]

File: src/ui/qgisapp.ui [hunk 1]
```diff
@@ -1886,7 +1886,7 @@
    </property>
    <property name="toolTip">
     <string>Moves a Label or Diagram
-Acts on all editable layers.</string>
+Acts on all layers.</string>
```
Comment (m-kuhn) [inline src/ui/qgisapp.ui]: What is the added value of leaving this message? Isn't it just stating the obvious and it would better be removed?

Comment (nyalldawson) [reply src/ui/qgisapp.ui]: Some of the labeling tools operate on the active layer only -- those also have an explanation in their tooltip reflecting this.

Comment (m-kuhn) [reply src/ui/qgisapp.ui]: We can also do that, but then we'll have to do it for all the feature tools etc. too I think?

Comment (nyalldawson) [reply src/ui/qgisapp.ui]: Sounds like a good idea in general to me!

**Pull request merged**